Library/Homebrew/cmd/bottle.rb

```diff
@@ -148,19 +148,19 @@ def bottle_formula(f)
       return ofail "Formula has no stable version: #{f.full_name}"
     end
 
-    if ARGV.include? "--no-revision"
-      bottle_revision = 0
+    if ARGV.include? "--no-rebuild"
+      rebuild = 0
     elsif ARGV.include? "--keep-old"
-      bottle_revision = f.bottle_specification.revision
+      rebuild = f.bottle_specification.rebuild
     else
-      ohai "Determining #{f.full_name} bottle revision..."
+      ohai "Determining #{f.full_name} rebuild..."
```

Comment:
I think this could be a tad clearer if it was “bottle rebuild” instead of just “rebuild”.